spec/features/checkouts_spec.rb

```diff
@@ -1,9 +1,34 @@
 require 'rails_helper'
 
 RSpec.feature "Checkouts", type: :feature do
+  before(:each) do
+  O1 = Order.create(Status: "Completed", total: 23600,
+                    vat: 2100, delivery_cost: 500, user_id: 6, 
+                    created_at: "2015-10-20 15:49:28",
+                    updated_at: "2015-10-20 15:49:28",
+                    transaction_id: "1VX775486T314881H",
+                    invoice: "", pickup_time: 26)
+  
+  O2 = Order.create(Status: "Completed", total: 23600, 
```

Comment:
Separate every 3 digits in the integer portion of a number with underscores(_).<br>Trailing whitespace detected.